src/rendition-mixin.js

```diff
@@ -66,6 +66,12 @@ class Representation {
 
     this.bandwidth = playlist.attributes.BANDWIDTH;
 
+    if (playlist.attributes.CODECS) {
+      this.codecs = playlist.attributes.CODECS;
+    }
```

Comment:
Make renditions much better to use. Now we know the playlist codecs and we can use `masterPlaylistController.fastQualityChange` or `masterPlaylistController.smoothQualityChange`